examples/c++/ExampleAdd.cpp

```diff
@@ -54,15 +57,27 @@ int main() {
   block_builder.create<mlir::stablehlo::AddOp>(loc, arguments, attributes)
       .getOperation();
 
-  mlir::Operation* op =
-      block_builder
-          .create<mlir::stablehlo::AddOp>(loc, arg0, arguments, attributes)
-          .getOperation();
+  mlir::Operation* op = block_builder
+                            .create<mlir::stablehlo::AddOp>(
+                                loc, tensorType, arguments, attributes)
+                            .getOperation();
   block_builder.create<mlir::func::ReturnOp>(loc, op->getResult(0));
 
-  // verify the module and dump
+  /** verify and dump the module **/
   assert(mlir::succeeded(mlir::verify(module.get())));
-  module->dump();
 
-  return 0;
+  /* interpret the  with concrete inputs **/
+  auto inputValue1 = mlir::DenseElementsAttr::get(
+      tensorType, block_builder.getFloatAttr(tensorType.getElementType(),
+                                             static_cast<double>(10)));
+  auto inputValue2 = mlir::DenseElementsAttr::get(
+      tensorType, block_builder.getFloatAttr(tensorType.getElementType(),
+                                             static_cast<double>(20)));
+  auto expectedValue = mlir::DenseElementsAttr::get(
+      tensorType, block_builder.getFloatAttr(tensorType.getElementType(),
+                                             static_cast<double>(30)));
```

Comment:
Nit: extract a function to reduce duplication? Alternatively, I think it may be possible to just write `mlir::DenseElementsAttr::get(tensorType, 30.0)`.

Reply:
Extracted a function. PTAL.